### PR TITLE
OCanren, GT 0.3.0 (non alpha)

### PR DESCRIPTION
**Pull request by Kakadu**
Signed-off-by: Kakadu <Kakadu@pm.me>

### CODE REVIEW

File: packages/OCanren-ppx/OCanren-ppx.0.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  "JetBrains Research https://research.jetbrains.org"
+]
+license: "LGPL-2.1"
+homepage: "https://github.com/JetBrains-Research/OCanren"
+bug-reports: "https://github.com/JetBrains-Research/OCanren/issues"
+depends: [
+  "dune" {>= "2.9" }
+  "ocaml" {>= "4.10"}
+  "dune-configurator"
+  "ppxlib" {>= "0.22" & <= "0.24"}
```
Comment (mseri) [inline packages/OCanren-ppx/OCanren-ppx.0.3.0/opam:36]: This prevents installation with ocaml 4.14. Is that expected?

File: packages/noCanren/noCanren.0.3.0/opam [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  "https://github.com/Kakadu"
+  "https://github.com/dboulytchev"
+]
+homepage: "https://github.com/Lozov-Petr/noCanren"
+bug-reports: "https://github.com/Lozov-Petr/noCanren/issues"
+license: "LGPL-2.1"
+
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.13" & <= "4.14" }
```
Comment (mseri) [inline packages/noCanren/noCanren.0.3.0/opam:16]:
```suggestion
  "ocaml" {>= "4.13" & < "4.14" }
```
Should this be `<`?

File: packages/OCanren-ppx/OCanren-ppx.0.3.0/opam (hunk 1 above)
Comment (Kakadu) [reply packages/OCanren-ppx/OCanren-ppx.0.3.0/opam]: It's installable on 4.14 from now

**Comment by Kakadu** [comment]
@mseri, ready for review.

File: packages/OCanren-ppx/OCanren-ppx.0.3.0/opam [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+run-test:  []
```
Comment (mseri) [inline packages/OCanren-ppx/OCanren-ppx.0.3.0/opam:56]:
```suggestion
```

**Comment by mseri** [comment]
Thanks!

**Pull request merged**